test-framework/kubernetes-client/pom.xml

```diff
@@ -21,10 +21,6 @@
             <groupId>io.fabric8</groupId>
             <artifactId>kubernetes-server-mock</artifactId>
             <exclusions>
-                <exclusion>
-                    <groupId>com.squareup.okhttp3</groupId>
-                    <artifactId>okhttp</artifactId>
-                </exclusion>
```

Comment:
Why did you remove this one?

Reply:
As a consequence of refactoring of the kubernetes client dependencies, otherwise the tests would fail due to missing classes.

Reply:
Excluding that dependency results in failures like
````
Caused by: java.lang.NoClassDefFoundError: okio/Sink
	at io.fabric8.kubernetes.client.server.mock.KubernetesServer.before(KubernetesServer.java:69)
	at io.quarkus.test.kubernetes.client.KubernetesServerTestResource.initServer(KubernetesServerTestResource.java:39)
	at io.quarkus.test.kubernetes.client.AbstractKubernetesTestResource.start(AbstractKubernetesTestResource.java:25)
	at io.quarkus.test.common.TestResourceManager.lambda$start$1(TestResourceManager.java:105)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.ClassNotFoundException: okio.Sink
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
````